fix_android_dependencies.py

```diff
@@ -107,5 +107,35 @@ def update_all():
         with open(gradle_file, 'w') as f:
             f.write(new_data)
 
+def update_all():
+    """Runs through all build.gradle files and performs replacements."""
+
+    project_root = os.getcwd()
+    print 'Repo root: {}'.format(project_root)
+
+    top_level_report = project_root + RELATIVE_PATH_TO_JSON_REPORT
```

Comment:
For adding paths together we should use `os.path.join()` instead of string operations.